crm_autoalias/__openerp__.py

```diff
@@ -0,0 +1,37 @@
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
```

Comment:
Shorten the headers as https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py